packages/v-pool/package.json

```diff
@@ -35,6 +35,6 @@
     "pg-cursor": "^1.3.0"
   },
   "peerDependencies": {
-    "vertica-nodejs": ">=8.0"
+    "vertica-nodejs": "0.9.0-pre.1"
```

Comment:
I've only ever seen a pre-release string of alpha or beta, do we want to use either of those instead of pre?

Reply:
pre and rc are also used, according to Alexey.